src/util/isEventTarget.ts

```diff
@@ -0,0 +1,3 @@
+export function isEventTarget(sourceObj: any): sourceObj is EventTarget {
```

Comment:
I seem we don't have usage other than `fromEvent`.

When I wrote this patch, I thought we should add them as a utility function which can be used from other modules for organizing a code. But now I feel it may be "premerture" refactoring. I'll merge their functions into `fromEvent`.